exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/Foreman.java

```diff
@@ -541,9 +546,9 @@ private void runPreparedStatement(final PreparedStatementHandle preparedStatemen
           ServerPreparedStatementState.PARSER.parseFrom(preparedStatementHandle.getServerInfo());
     } catch (final InvalidProtocolBufferException ex) {
       throw UserException.parseError(ex)
-          .message("Failed to parse the prepared statement handle. " +
-              "Make sure the handle is same as one returned from create prepared statement call.")
-          .build(logger);
+      .message("Failed to parse the prepared statement handle. " +
```

Comment:
The initial indentation was better, please revert this change.

Reply:
Ok.